xbmc/network/upnp/UPnPPlayer.h

```diff
@@ -25,7 +25,7 @@ namespace UPNP
 
 class CUPnPPlayerController;
 
-class CUPnPPlayer : public IPlayer, public CThread, public IRenderLoop
```

Comment:
Can `#include "guilib/DispResource.h"` removed now that this class is no longer derived from `IRenderLoop`?

Reply:
oops, leftover :)